src/main/java/com/salesforce/dev/framework/utils/DataDrivenManager.java

```diff
@@ -148,4 +149,25 @@ public DataDrivenManager(){
         }
         return chattersArray.iterator();
     }
-}
+
+    public Iterator<Object[]> getObjects(String nameJson, Class<?> elementClass) {
```

Comment:
You need to apply this method about the duplicated methods.
so you need to remove getChatter, getCampaign, ....... etc
and also this method should be static